int32 type
int32 id
em_planner_ros/TrajectoryPoint traj_p
float64 width
float64 length
float64 height
float64 confidence
float64 age

